packages/front_end/Dockerfile.dev

```diff
@@ -17,7 +17,7 @@ RUN apt-get update
 RUN apt-get install libgl1 -y
 
 
-RUN cd ./front_end/app && npm i && cd ../
+RUN cd ./front_end/app && npm i --force && cd ../
```

Comment:
this is specifically because the react-facebook-login package explicitly requires the force flag - specified in their documentation